pynopegl-utils/pynopegl_utils/viewer/__init__.py

```diff
@@ -291,22 +291,35 @@ def _export_video(self, filename: str, res_index: int, profile_index: int, sampl
 
             fd_r, fd_w = os.pipe()
 
+            exe = ["ffmpeg"]
+            input = ["pipe:%d" % fd_r]
+
+            if platform.system() == "Windows":
+                import msvcrt
+
+                handle = msvcrt.get_osfhandle(fd_r)
+                os.set_handle_inheritable(handle, True)
+                input = [f"handle:{handle}"]
+                exec = [sys.executable, "-m", "pynopegl_utils.viewer.ffmpeg"]
```

Comment:
can we name `ffmpeg` to `ffmpeg_win32` or something?

Also, `ffmpeg` or `ffmpeg_bin` sounds better than `exec` to me

Reply:
Done.